contrib/devtools/security-check.py

```diff
@@ -145,7 +145,6 @@ def check_ELF_separate_code(executable):
     EXPECTED_FLAGS = {
         # Read + execute
         '.init': 'R E',
-        '.plt': 'R E',
```

Comment:
Is `.plt` writable?
We should probably make this specific on architecture. If POWER9 is an exception here that needs the jump table to be writable (which would be kind of insecure in itself), let's make a specific exception for that.

Reply:
Agree with @laanwj. No need to wholesale relax our checks just to accommodate POWER. A special case with a comment should be fine.

Reply:
Fixed